aten/src/ATen/Config.h.in

```diff
@@ -10,3 +10,4 @@
 #define AT_MKL_ENABLED() @AT_MKL_ENABLED@
 #define AT_NNPACK_ENABLED() @AT_NNPACK_ENABLED@
 #define CAFFE2_STATIC_LINK_CUDA() @CAFFE2_STATIC_LINK_CUDA_INT@
+#define AT_BLAS_ENABLED() @USE_BLAS@
```

Comment:
Sorry, this conflicts with internal caffe2 define, can you please rename AT_BLAS_ENABLED to something else.